packages/server/src/api/todo/social/getFeed.ts

```diff
@@ -36,6 +36,7 @@ export default function (server: Server) {
           )
           .leftJoin(User, 'author', 'author.did = post.creator')
           .where('user.did = :did', { did: requester })
+          .andWhere('post.creator != :requester', { requester })
```

Comment:
we weren't returning a user's posts in their feed, but we were returning them if they were reposted, so this strips those out as well. can change this if we like

Reply:
Actually I think we need user's posts in their feed, but not the retweets? It can be confusing not to see your own posts

Reply:
Made an issue. i'll get to it soonish: https://github.com/bluesky-social/adx/issues/201